pkgs/development/python-modules/stdlibs/default.nix

```
@@ -7,7 +7,7 @@
 
 buildPythonPackage rec {
   pname = "stdlibs";
-  version = "2022.10.9";
+  version = "2023.11.2";
   format = "pyproject";
 
   disabled = pythonOlder "3.7";
```

https://github.com/omnilib/stdlibs/blob/a2f0bbf6198589218abe8d38878acafc2e89af21/pyproject.toml#L20
```suggestion
  disabled = pythonOlder "3.8";
```

---

Thanks for checking.